app/ui.R

```diff
@@ -13,13 +13,52 @@ aboutDashboardText <- includeMarkdown("assets/about-dashboard.md")
 # Layout
 ########
 
+create_output_panel <- function(title, suffix) {
+  tabPanel(title,
+    value = paste0("evaluations", suffix),
+    fluidRow(column(11, textOutput(paste0("renderWarningText", suffix)))),
+    plotlyOutput(outputId = paste0("summaryPlot", suffix), height = "auto"),
+    fluidRow(
+      column(11,
+        offset = 1,
+        hidden(div(id = paste0("refresh-colors", suffix), actionButton(inputId = paste0("refreshColors", suffix), label = "Recolor")))
+      )
+    ),
+    tags$br(),
+    plotlyOutput(outputId = paste0("truthPlot", suffix), height = "auto"),
+    fluidRow(
+      column(11,
+        offset = 1,
+        div(id = paste0("data-loading-message", suffix), "DATA IS LOADING...(this may take a while)"),
+        hidden(div(id = paste0("truth-plot-loading-message", suffix), "Fetching 'as of' data and loading observed values...")),
+        hidden(div(id = paste0("notes", suffix), "About the Scores")),
+        hidden(div(
+          id = paste0("scoreExplanations", suffix),
+          hidden(div(id = paste0("wisExplanation", suffix), wisExplanation)),
+          hidden(div(id = paste0("sharpnessExplanation", suffix), sharpnessExplanation)),
+          hidden(div(id = paste0("aeExplanation", suffix), aeExplanation)),
+          hidden(div(id = paste0("coverageExplanation", suffix), coverageExplanation))
```

Comment:
question: Do we still need coverage? we dropped it from `expectedCols`

Reply:
I changed the `expectedCols` code in `app/R/data.R` to make it clearer that we're _not_ dropping the coverage columns, just renaming them.